recipes/pyflamegpu/meta.yaml

```
@@ -0,0 +1,80 @@
+{% set name = "pyflamegpu" %}
+{% set version_py = "2.0.0rc1" %}
+{% set version_cpp = "2.0.0-rc1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version_py }}
+
+source:
+  # Temporary, will switch to release, when conda recipe is working
+  url: https://github.com/FLAMEGPU/FLAMEGPU2/archive/refs/heads/rob_conda.tar.gz
+  # url: https://github.com/FLAMEGPU/FLAMEGPU2/archive/refs/tags/{{ version_cpp }}.tar.gz
+
+build:
+  number: 0
+  script_env:
+    - FLAMEGPU_CONDA_BUILD_THREADS
+    - FLAMEGPU_CONDA_CUDA_ARCHITECTURES
+  missing_dso_whitelist:
+    - $RPATH/libcuda.so.1  # [linux] Ignore as this comes from CUDA driver
+    - $RPATH/libnvrtc.so.12  # [linux] Conda-build can't find this in run reqs cuda-cudart
+    - $RPATH/libcudart.so.12  # [linux] Conda-build can't find this in run reqs cuda-nvrtc
+    - $RPATH/nvrtc64_120_0.dll  # [win64] Conda-build can't find this in run reqs cuda-cudart
+    - $RPATH/cudart64_12.dll  # [win64] Conda-build can't find this in run reqs cuda-nvrtc
+  ignore_run_exports:
+    - python  # [linux] Not clear why conda thinks this isn't used
+    - astpretty  # [linux, win64] Not clear why conda thinks this isn't used
+    - vc14_runtime  # [win64] Not clear why conda thinks this isn't used (we aren't static linking)
+    - ucrt  # [win64] Not clear why conda thinks this isn't used (we aren't static linking)
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}
+    - cuda-cudart-dev  # [linux]
+    - cuda-nvrtc-dev  # [linux]
+    - libcurand-dev  # [linux]
```

```suggestion
  {% if cuda_major == 12 %}
    - cuda-cudart-dev  # [linux]
    - cuda-nvrtc-dev  # [linux]
    - libcurand-dev  # [linux]
  {% endif %}
```
These packages are only for CUDA >= 12. Our package structure changed for CUDA 12. Before that, everything was shipped as one monolithic cudatoolkit package.

Also, these are libraries, so they belong in the HOST requirements section.